src/Apps/Order/redirects.tsx

```diff
@@ -258,6 +260,7 @@ graphql`
     internalID
     mode
     state
```

Comment:
**question:** Can we remove this field, or does Force still need to know about it?

Reply:
I forgot to check the tests before writing this comment - it makes sense now that you're using both `state` and `displayState`.